### ISSUE TITLE
Instance: The "Sending a new message" label is insufficiently descriptive | WCAG 2.2 Violation - There are non-descriptive/insufficiently descriptive labels present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#34]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

#### Further information:

1. 

### Screenshot

![Screenshot of the 'Sending a new message' labelling element.](https://github.com/user-attachments/assets/02e68fa5-7005-4dac-b582-4eec6e38a6f4)

### Code Reference

```
<form id="chat-commentform" class="chat-form">
	<div class="chat-reply-indicator">
		<div id="chat-new-message-info" class="chat-reply-indicator-text">
			<span>Sending a new message</span>
		</div>
	</div>

	<div class="chat-form-logged-in">
		<div class="chat-form-input-wrapper">
			<textarea id="comment" name="comment" aria-describedby="chat-new-message-info" placeholder="Type your message... (use @ to mention)" rows="1" required="" style="height: 45px;"></textarea>
			[…]
		</div>
		[…]
	</div>
</form>
```
